### ISSUE TITLE
Enabling Telegram on Windows Crashes Pidgin

### ISSUE THREAD

**Issue by redyoshi49q** [majn/telegram-purple#324]
When starting a Pidgin session with a Telegram account enabled (or when running Pidgin with the -n parameter from the command line, disabling Telegram, setting the connection status to online, and enabling Telegram again), Pidgin crashes with the following error popup:

	Microsoft Visual C++ Runtime Library

	This application has requeste the Runtime to terminate it in an unusual way.
	Please contact the application's support team for more information.

I'm using telegram-purple-1.3.0rev2.exe (the newest available for Windows as of this issue report) and Pidgin 2.10.11.  The plugin was previously working with this version of Pidgin; there may have been a change in the Telegram protocol that broke the plugin in the past month or so.

Below is the terminal output from running Pidgin (with the -n parameter) from a Cygwin command line.  After Pidgin successfully started, the session's Telegram account was disabled, the status was set to online, and the Telegram account was enabled again (at which point Pidgin crashed).  While the output references C:\cygwin\tmp\pidgin.RPT, no such file exists after the crash, and the standard C:\Users\Ethan\AppData\Roaming\\.purple\pidgin.RPT is not edited by the crash, with or without the use of Cygwin.

	1262/4086 MB RAM 0.00 0.00 0.00 1/16 Sat, Nov  5, 2016 ( 1:41:43am CDT) [0 jobs]
	[ethan@firetail: +2] /cygdrive/c/Program Files (x86)/Pidgin $ ./pidgin.exe -n
	Loaded exchndl.dll
	 Setting exchndl.dll LogFile to C:\cygwin\tmp\pidgin.RPT
	 Setting exchndl.dll DebugInfoDir to C:\Program Files (x86)\Pidgin\pidgin-2.10.11-dbgsym
	GTK+ path found: C:\Program Files (x86)\Pidgin\Gtk\bin
	Using SetDllDirectory
	Setting locale: LANG=en
	Looking for Perl... Could not open reg subkey: SOFTWARE\Perl
	Error: (2) The system cannot find the file specified.

	not found.
	Looking for MIT Kerberos... Could not open reg subkey: SOFTWARE\MIT\Kerberos
	Error: (2) The system cannot find the file specified.

	not found.

	(Pidgin:27052): Json-CRITICAL **: json_object_get_array_member: assertion `node != NULL' failed

	(Pidgin:27052): Json-CRITICAL **: json_array_get_length: assertion `array != NULL' failed

	(Pidgin:27052): Json-CRITICAL **: json_object_get_string_member: assertion `node != NULL' failed

	(Pidgin:27052): Json-CRITICAL **: json_object_get_string_member: assertion `node != NULL' failed

	(Pidgin:27052): Json-CRITICAL **: json_object_get_string_member: assertion `node != NULL' failed

	(Pidgin:27052): Json-CRITICAL **: json_object_get_string_member: assertion `node != NULL' failed

	(Pidgin:27052): Json-CRITICAL **: json_object_get_string_member: assertion `node != NULL' failed

	(Pidgin:27052): Json-CRITICAL **: json_object_get_string_member: assertion `node != NULL' failed

	(Pidgin:27052): Json-CRITICAL **: json_object_get_string_member: assertion `node != NULL' failed

	(Pidgin:27052): Json-CRITICAL **: json_object_get_int_member: assertion `node != NULL' failed

	(Pidgin:27052): Json-CRITICAL **: json_object_get_int_member: assertion `node != NULL' failed
	Prefix='C:\Program Files (x86)\Pidgin\spellcheck'
	Looking for modules in dir='C:\Users\Ethan\AppData\Local\enchant'
	Looking for modules in dir='C:\cygwin\home\ethan\enchant'
	Looking for modules in dir='C:\Program Files (x86)\Pidgin\spellcheck\lib\enchant'
	Assertion failed!

	Program: C:\Program Files (x86)\Pidgin\pidgin.exe
	File: queries.c, Line 119

	Expression: c
	1311/4086 MB RAM 0.00 0.00 0.00 1/16 Sat, Nov  5, 2016 ( 1:44:46am CDT) [0 jobs]
	[ethan@firetail: +2] /cygdrive/c/Program Files (x86)/Pidgin $

**Comment by BenWiederhake** [comment]
Looks like this is a duplicate of https://github.com/majn/telegram-purple/issues/220

**Comment by redyoshi49q** [comment]
@BenWiederhake That's entirely possible; similar output behavior appears at the point of the crash, and one of the Telegram groups I participate in sends an obscene amount of messages, including image, audio, and video attachments.  I scanned the recent open and closed tickets before submitting this bug, thinking it might have been caused by a change in the protocol; in retrospect, I should have searched for related bugs instead (my apologies).

I'm closing this bug; I've subscribed myself to #220.
